app/controller/settings.js

```diff
@@ -0,0 +1,51 @@
+var mongoose = require('mongoose');
+var Settings = mongoose.model('Settings');
+
+SettingsService = {
+/**
+     * Retrieves the Settings file from the DB. 
+     * @param req
+     * @param res
+     */
+    get : function(req, res){
+        Settings.findOneOrCreate({ name: 'Settings' }, function (err, response){
+            var result = {
+                status: 200,
+                message: response
+            };
+            if (err){
+                result.status = 500;
+                result.message = err;
+            } else if (!response){
+                result.status = 404;
+                result.message = err;
+            }
+            res.status(result.status).json(result.message);
+        });
+    },
+
+    /**
+     * Updates given Settings by Id.
+     * @param req
+     * @param res
+     */
+    update: function(req, res) {
+        var id = req.params.id;
+        Settings.update({ '_id': id }, req.body, { upsert: true }, function (err, response){
```

Comment:
I'm getting a `DeprecationWarning: collection.update is deprecated. Use updateOne, updateMany, or bulkWrite instead.` on this. Looks like `updateOne` should work for this.

Reply:
Fixed.